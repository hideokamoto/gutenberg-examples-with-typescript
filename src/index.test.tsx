import React from 'react'
import { create } from 'react-test-renderer'
import { BlockEditProps, BlockSaveProps } from '@wordpress/blocks';
import {
    Save, edit, BlockAttributes
} from './index'

describe("index.tsx", () => {
    beforeAll(() => {
        window.matchMedia = jest.fn()
    })
    describe('save', () => {
        let props: BlockEditProps<BlockAttributes> = {
            attributes: {
                content: 'content'
            },
            setAttributes: jest.fn(),
            className: '',
            isSelected: false,
        }
        it("should match snapshot", () => {
            expect(create(<Save {...props} />).toJSON()).toMatchSnapshot()
        })
    })
})