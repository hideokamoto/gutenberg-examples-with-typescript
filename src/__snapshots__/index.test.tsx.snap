// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.tsx save should match snapshot 1`] = `<save />`;
